src/open_producten/producttypen/admin/actie.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from open_producten.producttypen.models import Actie
+
+
+class ActieInline(admin.TabularInline):
+    model = Actie
+    extra = 1
+    ordering = ("pk",)
```

Comment:
This one doesn't seem to be used currently, was this intended to be added to the ProductTypeAdmin?